Makefile

```diff
@@ -205,7 +205,30 @@ clean:
 distclean: clean
 	-rm -rf vendor
 
+modules:
```

Comment:
For people using these new make targets, do we expect/require that they have set GO111MODULE=on in their own environment, or should the make commands be doing that?

Reply:
Good question - I think we should, but I couldn't figure out how to do that in a Makefile. For example, none of these work (note - all start-of-line space characters are tabs):

Input:
```
modules:
	GO111MODULE="on"
	-go mod tidy
```
Output:
```
kubedirector git:(add-go-mod) ✗ make modules
GO111MODULE="on"
go mod tidy
go: modules disabled inside GOPATH/src by GO111MODULE=auto; see 'go help modules'
make: [modules] Error 1 (ignored)
```

Input:
```
modules:
	set GO111MODULE="on"
	-go mod tidy
```
Output:
```
kubedirector git:(add-go-mod) ✗ make modules
set GO111MODULE="on"
go mod tidy
go: modules disabled inside GOPATH/src by GO111MODULE=auto; see 'go help modules'
make: [modules] Error 1 (ignored)
```

Input:
```
modules:
	export GO111MODULE="on"
	-go mod tidy
```
Output:
```
kubedirector git:(add-go-mod) ✗ make modules
export GO111MODULE="on"
go mod tidy
go: modules disabled inside GOPATH/src by GO111MODULE=auto; see 'go help modules'
make: [modules] Error 1 (ignored)
```

Reply:
```makefile
modules:
	-GO111MODULE=on go mod tidy
```